index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>ERNEST Portfolio Website</title>
+</head>
+<body>
+    <!-- defining the header and wraping it in divs and will contain the logo-->
+    <div class="main_container">
+        <div class="inside_container">
+            <header class="main_header">
+              <nav class="navbar">
+                <a href="#" class="my-portfolio-logo">
+                    <img src="./assets/MyLogo.png" alt="My-logo"/>
+                </a>
+                <a><img src="./assets/menu.png" alt="menu-icon"/></a>
+              </nav>
+            </header>
+    <!-- Defining the main section for the description of the developer -->
+            <section class="main_section">
+                <div class="container">
+                  <h1 class="h1">I am Ernest Lasten. Glad to see you!</h1>
+                    <p class="main_content">
+                        I’m a software developer! I can help you build a product ,
+                        feature or website Look through some of my work and experience!
+                        If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
+                    <div class="connect_links">
+                        <a class="connect_text">LET'S CONNECT</a>
+                        <ul class="links_list">
+                            <li class="media_links">
+                                <a href="#"><img src="./assets/twitterIcon.svg" alt="icon for twitter"/></a>
+                            </li>                            
+                            <li class="media_links">
+                                <a href=""><img src="./assets/linkedInIcon.png" alt="icon for linkedIn"/></a>
+                            </li>
+                            <li class="media_links">
+                                <a href=""><img src="./assets/mIcon.svg" alt="icon for Medium"/></a>
+                            </li>
+                            <li class="media_links">
+                                <a href=""><img src="./assets/gitHubIcon.svg" alt="icon for GitHub"/></a>
+                            </li>
+                            <li class="media_links">
+                                <a href=""><img src="./assets/peaceIcon.svg" alt="icon for M"/></a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </section>
```

Comment:
As you can see, the background for the main section is broken and is not visible. Please have a look at this and try to fix it. Here is a snapshot for your reference.
![snapshot](https://user-images.githubusercontent.com/33268927/170635711-3a5a4eaa-f184-487c-aec1-f98e425f76b7.png)